src/bundle/Controller/ContentOnTheFlyController.php

```diff
@@ -198,11 +230,15 @@ public function editContentAction(
         string $languageCode,
         int $contentId,
         int $versionNo,
-        int $locationId
+        ?int $locationId
     ) {
         $content = $this->contentService->loadContent($contentId, [$languageCode], $versionNo);
         $versionInfo = $content->getVersionInfo();
-        $location = $this->locationService->loadLocation($locationId);
+
+        $location = null;
+        if (!empty($locationId)) {
```

Comment:
It is worth to perform more strict check, Maybe `locationId !== null`

Reply:
@Nattfarinn please address it as a followup

Reply:
Fixed here: https://github.com/ezsystems/ezplatform-admin-ui/pull/1588